tests/quick_tests/CMakeLists.txt

```diff
@@ -92,7 +92,7 @@ FOREACH(_build ${DEAL_II_BUILD_TYPES})
 ENDFOREACH()
 
 # Test whether thread affinity is well behaved
-IF (DEAL_II_WITH_THREADS)
+IF (DEAL_II_WITH_TBB)
```

Comment:
Can't we always run it?

Reply:
you are right, this should always work. Let's see if that is true even under windows.